### DISCUSSION TITLE
Add Chris Titus Script repository?

### DISCUSSION THREAD

**Discussion by quadcom** [Devolutions/UniGetUI#2913] [General]
Is it possible to add the Chris Titus repository to his all-in-one Windows cleanup script and have it updated when a revision is pushed out?

**Comment by marticliment** [comment]
Is this script available from any package manager?

**Comment by quadcom** [comment]
I honestly don't know. THe normal install method is simply running a powershell command.

https://github.com/christitustech/winutil

**Comment by marticliment** [comment]
I am afraid this is out of scope unless the script is integrated with a package manager

**Comment by quadcom** [comment]
Hey @ChrisTitusTech, is there anything you can do to bring your scripts here?

**Comment by Lockszmith-GH** [comment]
In a way, the powershell script is a currated front end for winget.
UniGetUI is a complete UI - you can use them in parallel, but there are competing paradigms.

If you know what you want to install, just type the package name in UniGetUI and install it, you can even manage bundles with it.

I'm unsure of what you're asking to be done here.
If what you want is a method of installing the script via UniGetUI, then you need to find a way to incorporate the installation script in winget or scoop, and then it will be discoverable in UniGetUI.

Remember, UniGetUI is a frontend to other package managers, it does not manage package on it's own.

**Reply by quadcom** [reply]
I posted the question here for two reasons. First, the script is awesome. Sure there's some overlap with installing apps from packages but it's the crap removal and telemetry disabling that it excels at. In Addition there's the tiny Win11 builder built in.

The problem is I can never remember the URL to run it so I always have to go search or it again and copy/paste to get it up and running.

UniGet gets installed and runs in the BG keeping track of stuff. The script isn't something you would just run once and be done. Everytime Win has some updates it re-enables stuff you had turned off beforehand -the new Recall feature being a prime example.

I just figured it would be something that would help everyone. If it couldn't be included in the usual way, perhaps some kind of colab or another new section that could coallate various scripts from the community(ies).

Just throwing ideas out there, maybe one of them sticks.

**Reply by Lockszmith-GH** [reply]
The way I see it, there are a couple of 'easy' routes here - none of them are UniGetUI specific:
- Submit a PR to [ScoopInstaller Extras bucket repo](https://github.com/ScoopInstaller/Extras) with the script's install.
- Ask ChrisTitusTech to package it into a winget installable option. (This can be done by a third party as well, but it is best maintained by the maintainer)

**Comment by ChrisTitusTech** [comment]
Just getting around to replying to old mentions... 2 years later, but UnigetUI is amazing work and it is added!

**Reply by J-HS-L** [reply]
@ChrisTitusTech You mean your script repo is now winget or pwsh7 pkg manager facing, and hence usable via unigetui?
